### DISCUSSION TITLE
How connect client to my VpnHood Server

### DISCUSSION THREAD

**Discussion by The-TRENERA** [vpnhood/VpnHood#37] [Q&A] [answered]
I create VpnHood Server. What data should I give to client (my friend) to join.
And where exactly should he enter the data (accessID key or what..).?.

**Comment by trudyhood** [comment, marked as answer]
Hi, 
If you are using Linux the run the following command:

`sudo dotnet /opt/VpnHoodServer/launcher/run.dll gen`

it gives you an access key started with vh://
Give it to your clients and let me know if there is any issue.

Best,
